.buildkite/heartbeat/heartbeat-pipeline.yml

```diff
@@ -324,12 +353,13 @@ steps:
 
   - group: "Packaging"
     key: "heartbeat-packaging"
+
     steps:
       - label: ":ubuntu: Heartbeat: Packaging Linux"
         key: "heartbeat-package-linux"
         env:
           PLATFORMS: "+all linux/amd64 linux/arm64 windows/amd64 darwin/amd64 darwin/arm64"
-          SNAPSHOT: true
```

Comment:
why is this removed?

Reply:
This env var was used in Jenkins [for image pushing](https://github.com/elastic/beats/blob/e40a406e6c4270266c8bac5939b7516afc2ca47d/Jenkinsfile#L486). Since we don't push anything in this step, it's not relevant.